src/assets/js/dynamic-addon-cards.js

```diff
@@ -113,13 +113,13 @@
         } else {
           const { current_version, promoted } = addon;
 
-          if (!current_version || !current_version.files) {
+          if (!current_version || !current_version.file) {
```

Comment:
Do we still need this check if it's impossible for `current_version.file` to be falsey?